conjurapi/resource.go

```diff
@@ -0,0 +1,25 @@
+package conjurapi
+
+import (
+	"fmt"
+)
+
+func (c *Client) CheckPermission(resourceId, privilege string) (bool, error) {
+	req, err := c.router.CheckPermissionRequest(resourceId, privilege)
+	if err != nil {
+		return false, err
+	}
+
+	resp, err := c.SubmitRequest(req)
+	if err != nil {
+		return false, err
+	}
+
+	if resp.StatusCode >= 200 && resp.StatusCode < 300 {
```

Comment:
I would have thought this would always be `200` if the permission exists. What situations would make the response code be 200<x<300?

Reply:
For example, 204 No Content.